### ISSUE TITLE
Jenkins display names aren't used

### ISSUE THREAD

**Issue by rjrjr** [AdamNowotny/BuildReactor#56]
Our Jenkins builds have heinous proper names, so we use it's Display Name feature to make them more palatable for wall displays. BuildReactor doesn't seem to notice this. I haven't figured out yet where this is reported in json, I'll update when I do. 

Happy to put up a PR if you could point me in the right direction.

![screen shot 2016-10-04 at 5 47 58 pm](https://cloud.githubusercontent.com/assets/1884445/19097597/d165877e-8a5a-11e6-860d-e8ed8817c126.png)

**Comment by AdamNowotny** [comment]
Looking at https://github.com/AdamNowotny/BuildReactor/blob/master/src/core/services/jenkins/jenkinsBuild.js the name should be just "displayName". Not sure if that's what the plugin uses. Can you check response from http://<jenkins>/job/<job_id>/api/json ?

**Comment by rjrjr** [comment]
Yeah, sorry about the misleading report. This is clearly about the Walldisplay plugin. Here's the relevant bit from the api call you asked for.

```
  "property": [
    {},
    {
      "notifyAborted": false,
      "notifyBackToNormal": false,
      "notifyFailure": false,
      "notifyNotBuilt": false,
      "notifySuccess": false,
      "notifyUnstable": false,
      "room": "",
      "startNotification": false
    },
    {
      "notifyAborted": true,
      "notifyBackToNormal": true,
      "notifyFailure": true,
      "notifyNotBuilt": true,
      "notifySuccess": true,
      "notifyUnstable": true,
      "room": "",
      "startNotification": true
    },
    {},
    {
      "wallDisplayBgPicture": null,
      "wallDisplayName": "Nice Name Here"
    }
  ],
```

**Comment by AdamNowotny** [comment]
Where is the "property" field in the json tree, same level as id, url, action etc.?
